src/odemis/acq/path.py

```diff
@@ -552,7 +549,13 @@ def _doSetPath(self, path, detector):
                                     self._stored[comp_role, axis] = comp.position.value[axis]
                                 break
                         else:
-                            logging.debug("Choice %s is not present in %s axis", pos, axis)
+                            if mode == "mirror-align" and pos == "pass-through":
+                                # That's typically a sign that the microscope file is incorrect
+                                # eg, a typo in the filter name, as if there is a filter wheel
```

Comment:
I think the code is correct, but the comment is really confusing. Make it 2 sentences -> `If there is a filter wheel and a ccd .....`

Reply:
Line 542: I cannot comment on this line, but the comment is not really helping as it describes what the code does. Would be more helpful to say something like `Handle the filter wheel in a special way. Search for the fw position that corresponds to the requested position (band). `